util/src/main/java/com/google/cloud/hadoop/util/CredentialFactory.java

```diff
@@ -254,131 +250,92 @@ public static Credential getCredentialFromMetadataServiceAccount()
    * href="https://code.google.com/p/google-api-java-client/wiki/OAuth2#Service_Accounts" > OAuth2
    * Service Accounts</a>.
    *
-   * @param serviceAccountEmail Email address of the service account associated with the keyfile.
-   * @param privateKeyFile Full local path to private keyfile.
    * @param scopes List of well-formed desired scopes to use with the credential.
    * @param transport The HttpTransport used for authorization
-   * @deprecated Use properly configured {@ CredentialOptions} with {@link
-   *     CredentialFactory#getCredential(List<String>)}
    */
-  @Deprecated
-  public static Credential getCredentialFromPrivateKeyServiceAccount(
-      String serviceAccountEmail,
-      String privateKeyFile,
-      List<String> scopes,
-      HttpTransport transport,
-      String tokenServerUrl)
-      throws IOException, GeneralSecurityException {
+  public Credential getCredentialFromPrivateKeyServiceAccount(
+      List<String> scopes, HttpTransport transport) throws IOException, GeneralSecurityException {
     logger.atFine().log(
-        "getCredentialFromPrivateKeyServiceAccount(%s, %s, %s)",
-        serviceAccountEmail, privateKeyFile, scopes);
+        "getCredentialFromPrivateKeyServiceAccount(%s, %s) for '%s' from '%s'",
```

Comment:
Lets keep the log format consistent as above. method name style or human readable style, either one

Reply:
It's still consistent - number of method parameters changed, so I moved useful information outside of the method declaration in the log statement.